common/src/main/res/color/radiobutton_icon_color.xml

```diff
@@ -14,14 +14,7 @@
   ~ You should have received a copy of the GNU General Public License
   ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   -->
-<ripple xmlns:android="http://schemas.android.com/apk/res/android"
-    android:color="@color/list_view_btn_ripple"
-    xmlns:tools="http://schemas.android.com/tools">
-
-    <item android:id="@android:id/mask">
-        <shape android:shape="rectangle">
-            <corners android:radius="?attr/cornerRadius" tools:radius="10dp" />
-            <solid android:color="@android:color/white" />
-        </shape>
-    </item>
-</ripple>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:color="@color/white" android:state_selected="true"/>
+    <item android:color="@color/steel_gray_250" android:state_selected="false"/>
+</selector>
```

Comment:
This diff doesn't make sense, `rounded_ripple_transparent` was deleted since `rounded_ripple_background` can do the same thing.